packages/clerk-js/src/core/resources/OrganizationMembership.ts

```diff
@@ -41,8 +40,16 @@ export class OrganizationMembership extends BaseResource implements Organization
       search: convertPageToOffset({ ...retrieveMembershipsParams, paginated: true }) as any,
     })
       .then(res => {
+        if (!res?.response) {
+          return {
+            total_count: 0,
+            data: [],
+          };
+        }
+
+        // TODO: Fix typing
         const { data: suggestions, total_count } =
-          res?.response as unknown as ClerkPaginatedResponse<OrganizationMembershipJSON>;
+          res.response as unknown as ClerkPaginatedResponse<OrganizationMembershipJSON>;
```

Comment:
How come this is a necessary change ? I believe you are using this logic in more places, shall we update the rest as well ?